examples/curio/curio-server.py

```diff
@@ -190,6 +191,13 @@ def __init__(self, sock, root):
 
 
 if __name__ == '__main__':
-    kernel = Kernel()
-    kernel.add_task(h2_server(('', 8080), sys.argv[1]))
+    host = sys.argv[2] if len(sys.argv) > 2 else "localhost"
+    kernel = Kernel(with_monitor=True)
+    kernel.add_task(h2_server((host, 5000),
+                              sys.argv[1],
+                              "{}.crt.pem".format(host),
+                              "{}.key".format(host)))
```

Comment:
Really? I tried it (after renaming them, part of this change) and it worked. At least with curl it does. I can't check it now with a browser (since the cert is for localhost and now I'm remote).